contrib/build-push-ceph-container-imgs-arm64.sh

```diff
@@ -22,15 +22,17 @@ function install_docker {
 function build_ceph_imgs {
   echo "Build Ceph container image(s)"
   for ceph_release in "${CEPH_RELEASES[@]}"; do
-    make DAEMON_BASE_TAG=daemon-base:"$RELEASE"-"${ceph_release}"-centos-7-aarch64 DAEMON_TAG=daemon:"$RELEASE"-"${ceph_release}"-centos-7-aarch64 RELEASE="$RELEASE" FLAVORS="${ceph_release},centos-arm64,7" BASEOS_REPO=centos build
+    CENTOS_RELEASE=$(_centos_release ${ceph_release})
```

Comment:
Forgive my lack of BASH knowledge but:

1. If this is set at the top of the script, will `CENTOS_RELEASE` carry on through the rest of the script?
2. Is the `build_ceph_imgs` function *always* called?  Just want to make sure we don't accidentally end up with an empty `CENTOS_RELASE`

Reply:
> 1. If this is set at the top of the script, will `CENTOS_RELEASE` carry on through the rest of the script?

True, unless I set `CENTOS_RELEASE` as a local variable for each function. But this shouldn't be a problem because we always call the `_centos_release` function before using that variable so we shouldn't get a `CENTOS_RELEASE` value from a previous call.

> 2. Is the `build_ceph_imgs` function _always_ called?

Not always, it depends if we're building devel containers or using the ceph-ci scenario.

https://github.com/ceph/ceph-container/blob/master/contrib/build-push-ceph-container-imgs.sh#L319-L320

> Just want to make sure we don't accidentally end up with an empty `CENTOS_RELASE`

This isn't possible because the `CENTOS_RELASE` variable is always set by the `_centos_release` function which always returns a value.